### PR TITLE
Add imaging-hub.data-commons.org

### PR DESCRIPTION
**Pull request by jawadqur**
Link to Jira ticket if there is one:

### CODE REVIEW

File: bih.data-commons.org/manifest.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    "environment": "bihprod",
+    "hostname": "bih.data-commons.org",
+    "revproxy_arn": "arn:aws:acm:us-east-1:533267425233:certificate/dcb6f91b-6e4c-4458-a9d7-3750ba92b754",
+    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/gtexdictionary/4.0.6/schema.json",
+    "dispatcher_job_num": "10",
+    "portal_app": "gitops",
+    "sync_from_dbgap": "False",
+    "kube_bucket": "kube_bucket.devplanetv1.gen3",
+    "logs_bucket": "logs-devplanetv1-gen3",
+    "useryaml_s3path": "s3://cdis-gen3-users/brh/user.yaml",
```
Comment (paulineribeyre) [inline bih.data-commons.org/manifest.json:44]:
```suggestion
    "useryaml_s3path": "s3://cdis-gen3-users/midrc-bih/user.yaml",
```

Comment (jawadqur) [reply bih.data-commons.org/manifest.json]: added

File: imaging-hub.data-commons.org/manifests/hatchery/hatchery.json [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    "local": true
+  },
+  "pay-models-dynamodb-table": "bmh-admin-portal-backend-bmhworkspacetableD4D73E43-34AAJOR7A5AG",
+  "sidecar": {
+    "cpu-limit": "1.0",
+    "memory-limit": "256Mi",
+    "image": "quay.io/cdis/ecs-ws-sidecar:2024.02",
+    "env": {
+      "NAMESPACE": "default",
+      "HOSTNAME": "brh.data-commons.org"
```
Comment (paulineribeyre) [inline imaging-hub.data-commons.org/manifests/hatchery/hatchery.json:20]: brh->bih

File: imaging-hub.data-commons.org/manifests/hatchery/hatchery.json [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+      "-s",
+      "/bin/sh",
+      "root"
+    ]
+  },
+  "nextflow-global": {
+    "sample-config-public-image": "public.ecr.aws/u5x5h6w3/nextflow-approved/public:gen3-amazonlinux-base-AL2023fix",
+    "imagebuilder-reader-role-arn": "arn:aws:iam::143731057154:role/nextflow-imagebuilder-reader"
+  },
+  "containers": [
```
Comment (paulineribeyre) [inline imaging-hub.data-commons.org/manifests/hatchery/hatchery.json:40]: let's just set an empty list of containers? i don't think we'll be using the workspace in BIH yet, and these options are too specific to BRH

File: imaging-hub.data-commons.org/portal/gitops.json [hunk 4]
```diff
@@ -0,0 +1,653 @@
...
+    "boardCounts": [],
+    "chartCounts": [],
+    "projectDetails": "boardCounts"
+  },
+  "components": {
+    "appName": "Biomedical Research Hub",
+    "index": {
+      "introduction": {
+        "heading": "",
+        "text": "The Biomedical Research Hub (BRH) is a standard-based data ecosystem for securely managing, analyzing and sharing biomedical data to support research communities and collaborations."
```
Comment (paulineribeyre) [inline imaging-hub.data-commons.org/portal/gitops.json:15]: this file has several references to BRH - but i thought we were using this https://github.com/uc-cdis/bih-data-commons for the portal config instead? can we just remove this one? i'm a bit confused about how this repo and the `bih-data-commons` repo intersect

Comment (jawadqur) [reply imaging-hub.data-commons.org/portal/gitops.json:15]: Thanks! We should update this file as well yeah. I'll see what I can do. 

FEFW is running on `/` and portal is running on `/portal`

File: imaging-hub.data-commons.org/manifests/hatchery/hatchery.json (hunk 3 above)
Comment (jawadqur) [reply imaging-hub.data-commons.org/manifests/hatchery/hatchery.json:40]: I think I'll remove hatchery configs for now.

File: imaging-hub.data-commons.org/manifest.json [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    "ambassador": "quay.io/datawire/ambassador:1.4.2",
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2024.02",
+    "aws-es-proxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/aws-es-proxy:v1.3.1",
+    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2024.02",
+    "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:2024.02",
+    "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.15.3-debian-cloudwatch-1.0",
+    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2.1.2",
+    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2024.02",
+    "guppy": "quay.io/cdis/guppy:feat_skip_disabled_fields",
+    "kayako-wrapper": "quay.io/cdis/kayako-wrapper-service:0.2.0",
```
Comment (paulineribeyre) [inline imaging-hub.data-commons.org/manifest.json:19]: we can remove kayako-wrapper, dashboard, hatchery, manifestservice, requestor, wts

File: imaging-hub.data-commons.org/manifest.json [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+      }
+    ],
+   "config_index": "default-commons-config-index"
+  },
+  "metadata": {
+    "USE_AGG_MDS": true,
+    "AGG_MDS_NAMESPACE": "bihprod"
+  },
+  "portal": {
+    "GEN3_BUNDLE": "ecosystem"
```
Comment (paulineribeyre) [inline imaging-hub.data-commons.org/manifest.json:73]: not sure this is right 🤷‍♀️

Comment (jawadqur) [reply imaging-hub.data-commons.org/manifest.json:73]: Hmm.. I'm actually not sure what this should be then?

Comment (paulineribeyre) [reply imaging-hub.data-commons.org/manifest.json:67]: looks like it's for "data hubs" that use the discovery page, maybe... https://github.com/uc-cdis/cdis-manifest/blob/d82f2a8ba0b49e2fce153d5e67ec3625854129a8/releases/2021/10/gen3_release_notes.md?plain=1#L126-L127
anyway this sounds fine

File: imaging-hub.data-commons.org/manifest.json [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2024.02",
+    "aws-es-proxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/aws-es-proxy:v1.3.1",
+    "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:2024.02",
+    "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.15.3-debian-cloudwatch-1.0",
+    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2024.02",
+    "guppy": "quay.io/cdis/guppy:feat_skip_disabled_fields",
+    "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:2024.02",
+    "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2024.02",
+    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:2024.02",
+    "requestor": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/requestor:2024.02",
```
Comment (paulineribeyre) [inline imaging-hub.data-commons.org/manifest.json:19]: just this one left to remove and i think we're good to go

File: imaging-hub.data-commons.org/metadata/aggregate_config.json [hunk 8]
```diff
@@ -0,0 +1,283 @@
+{
```
Comment (paulineribeyre) [inline imaging-hub.data-commons.org/metadata/aggregate_config.json:1]: oops i think we need to remove this too?

Comment (paulineribeyre) [reply imaging-hub.data-commons.org/metadata/aggregate_config.json:1]: the whole file

Comment (jawadqur) [reply imaging-hub.data-commons.org/metadata/aggregate_config.json]: Removed!

File: imaging-hub.data-commons.org/manifest.json (hunk 7 above)
Comment (jawadqur) [reply imaging-hub.data-commons.org/manifest.json]: Removed

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**